src/EventStore.Core/ClusterVNode.cs

```diff
@@ -1526,14 +1528,15 @@ public class ClusterVNode<TStreamId> :
 				}
 			}
 
+			var segmentConf = new Configuration(options.Application.SegmentWriteKey, flushAt:1, flushInterval:10, storageProvider: new InMemoryStorageProvider());
```

Comment:
there are two places we're constructing a Configuration object like this, do we need them both?

Reply:
I remember now. Initially, I intended to pass that configuration through dependency injection but it wasn't use by the info controller.